packages/edit-post/src/hooks/validate-multiple-use/index.js

```diff
@@ -44,8 +39,7 @@ const enhance = compose(
 		// Otherwise, only pass `originalBlockClientId` if it refers to a different
 		// block from the current one.
 		const blocks = select( blockEditorStore ).getBlocks();
-		const firstOfSameType = find(
-			blocks,
+		const firstOfSameType = blocks.find(
```

Comment:
`blocks` will always be an array:

https://github.com/WordPress/gutenberg/blob/64aadcd67dc4ec3df20f62cc2a2d1c3db7bb85eb/packages/block-editor/src/store/selectors.js#L173